woocommerce/woocommerce
# Remove not used feature flags

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR removes the following non-used feature flags

- product-variation-management
- product-virtual-downloadable
- product-external-affiliate
- product-grouped
- product-linked
- customer-effort-score-tracks

Closes #46844.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable the new product editor at `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`.
2. Go to Products > Add and do some smoke tests to verify that anything is broken.
3. Try the variations, downloadable products, change the product type, add custom fields, publish a couple of products, etc.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
